app/Models/Post.php

```diff
@@ -53,7 +53,22 @@ public function signup()
      */
     public function reactions()
     {
-        return $this->hasMany(Reaction::class);
+        // return $this->morphToMany(Reaction::class, 'postable');
+        return $this->morphMany(Reaction::class, 'postable');
+    }
```

Comment:
hm - right now we have `postable_id` and `postable_type` in the Reactions table - should this be changed to `reactionable_id` and `reactionable_type` and then in the model relations have `reactionable` instead of `postable`?